### DISCUSSION TITLE
The site is down for logged in users.

### DISCUSSION THREAD

**Discussion by Rounak-stha** [vercel/community#1771] [Help] [answered]
Down for about 10 minutes now.

![image](https://user-images.githubusercontent.com/66557682/211154750-e7cbe609-0027-4d7f-9420-c9cd5e76384c.png)

**Comment by amyegan** [comment, marked as answer]
Thanks for reporting this @Rounak-stha. Are you still experiencing this problem?

**Reply by Rounak-stha** [reply]
Now it's working as expected.
